### PR TITLE
Garden images

### LINKED ISSUE #96: As a developer, I reuse the Image service from Sweater Weather to generate images for Gardens

### PR DESCRIPTION
**Pull request by AngelaGuardia**
Added an Image Facade and Service to get custom images for gardens

**Changes Made**
- Added an image column to the gardens table in the schema that holds a String for the image url
- Added image as an attribute to be returned by the image serializer

**Notes**
- Used Pexels API to get images
- Made the image query based on the garden name
- Made image "selection" random from the results of that query. This may return some interesting results. If anyone has ideas on how to improve the selection let me know!

closes #96

### CODE REVIEW

**Review by dcoleman21 — APPROVED** [review]
This looks great @AngelaGuardia! The randomness of the images might be fun! We can figure that out I'm sure.

File: spec/requests/api/v1/garden_request_spec.rb [hunk 1]
```diff
@@ -97,6 +97,21 @@
...
+      post "/api/v1/gardens", headers: headers, params: JSON.generate(garden_params)
+
+      new_garden = Garden.last
+      expect(response).to be_successful
+      expect(new_garden.image).to be_a(String)
+
+      user_garden = UserGarden.last
+      expect(user_garden.garden_id).to eq(new_garden.id)
+    end
+
```
Comment (dcoleman21) [inline spec/requests/api/v1/garden_request_spec.rb:114]: What if the garden image doesn't return an image. We need to add a sad path test for if there is no image and set it to the default image that we are using there right now.

**Review by lkriffell — APPROVED** [review]
I like this! I think one way we could use this feature really well would be to display a random image on garden creation and a user can either click keep or go to the next random picture.

File: app/facades/image_facade.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ImageFacade
+  def self.new_image(name)
+    ImageService.new_image(name)
+  end
+end
```
Comment (dcoleman21) [inline app/facades/image_facade.rb:5]: If this comes back empty, set it to the default image

**Review by dcoleman21 — CHANGES_REQUESTED** [review]
I left some comments in the request spec and the facade in regards to setting a default image and sad path testing for the instances when no image is returned.

**Pull request merged**